craft_parts/executor/environment.py

```diff
@@ -162,6 +162,9 @@ def _get_global_environment(info: ProjectInfo) -> dict[str, str]:
         "CRAFT_PROJECT_DIR": str(info.project_dir),
         # Build system configurations for the project.
         "CARGO_HOME": str(info.dirs.work_dir / "cargo"),
+        "CRAFT_BACKSTAGE": str(info.backstage_dir),
```

Comment:
similar, this would enable an override-build behavior.

Reply:
I thought I had removed this... That explains the unexpected test breakages!